source/faq/questions/using-cypress-faq.md

```diff
@@ -311,35 +311,25 @@ It is possible to upload files in your application but it's different based on h
 
 ## {% fa fa-angle-right %} What is the projectId for?
 
-Once you {% url "set up your tests to record" dashboard-service#Recorded-runs %}, we generate a unique `projectId` for your project and automatically insert it into your `cypress.json` file.
-
-***The `projectId` is a 6 character string in your cypress.json:***
+The `projectId` is a 6 character string that helps identify your project once you've {% url "set up your tests to record" dashboard-service#Recorded-runs %}. It's generated by Cypress and typically is found in your {% url `cypress.json` configuration %} file.
 
 ```json
 {
   "projectId": "a7bq2k"
 }
 ```
 
-This is how we uniquely identify your project. If you manually alter this, *Cypress will no longer be able to identify your project or find the recorded tests for it*. We recommend that you check your `cypress.json`, including the `projectId`, into source control.
+For further detail see the {% url Identification dashboard-service#Identification %} section of the {% url "Dashboard Service" dashboard-service %} docs.
 
 ## {% fa fa-angle-right %} What is a Record Key?
 
-Once you're {% url "set up to record test runs" dashboard-service#Recorded-runs %}, we automatically generate a *Record Key* for the project.
-
-***A record key is a GUID that looks like this:***
+A *Record Key* is a GUID that's generated automatically by Cypress once you've {% url "set up your tests to record" dashboard-service#Recorded-runs %}. It helps identify your project and authenticate that your project is even *allowed* to record tests.
```

Comment:
This FAQ also duplicated the existing documentation from the _Dashboard Services_ page.

Again, I've tried to answer the question with a direct answer and a screenshot of where to find the _Record Key_ while also removing much of the duplication and directing them to the fuller documentation if they wanna dig in.

In making these changes I asked myself _"If I wanted to know what the `projectId`/Record Key were, would I be satisfied with these answers?"_... and yeah, I would. School me if they're too brief and/or don't work for you.

Reply:
Great, yes, this is preferable to what we had.